includes/Documents/OrderDocument.php

```diff
@@ -1477,6 +1478,59 @@ public function get_pdf() {
 		return apply_filters( 'wpo_wcpdf_get_pdf', $pdf, $this );
 	}
 
+	/**
+	 * Replace .avif images with .jpg images in HTML
+	 * @param string $html The HTML to process
+	 * 
+	 * @return string The processed HTML
+	 */
+	public function replace_avif_images( string $html ): string {
+		$replaced_images = array();
+
+		if ( ! empty( $html ) ) {
+			$dom = new \DOMDocument();
+			$dom->loadHTML( $html );
```

Comment:
Maybe you should clear errors here if HTML is malformed, see example: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/e8581e1c1484d8cf41e1e4817e2e70b6f322f8e3/wpo-ips-functions.php#L630-L632